src/node_platform.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SRC_NODE_PLATFORM_H_
+#define SRC_NODE_PLATFORM_H_
+
+#include "libplatform/libplatform.h"
+#include "uv.h"
+
+namespace node {
+
+class NodePlatform : public v8::Platform {
+ public:
+  NodePlatform(int thread_pool_size, uv_loop_t* loop,
+               v8::TracingController* tracing_controller);
+  virtual ~NodePlatform();
+
+  // v8::Platform implementation.
+  size_t NumberOfAvailableBackgroundThreads() override;
+  void CallOnBackgroundThread(v8::Task* task,
+                              ExpectedRuntime expected_runtime) override;
+  void CallOnForegroundThread(v8::Isolate* isolate, v8::Task* task) override;
+  void CallDelayedOnForegroundThread(v8::Isolate* isolate, v8::Task* task,
+                                     double delay_in_seconds) override;
+  bool IdleTasksEnabled(v8::Isolate* isolate) override;
+  double MonotonicallyIncreasingTime() override;
+  v8::TracingController* GetTracingController() override;
+
+ private:
+  int num_threads_;
+  uv_loop_t* loop_;
```

Comment:
I've made the thread count const. `uv_timer_init` and `uv_queue_work` don't like const loops.